Shared/Mobile-Rounds.ViewModels/Platform/IFileHandler.cs

```diff
@@ -9,6 +9,7 @@ namespace Mobile_Rounds.ViewModels.Platform
     public interface IFileHandler
     {
         Task<string> GetFileAsync(string fileName);
+        Task<TOutput> GetFileAsync<TOutput>(string fileName) where TOutput: new();
```

Comment:
Auto extracts the data into the format you want, rather than needing to do another parsing step.